aws_google_auth/google.py

```diff
@@ -133,7 +138,7 @@ def parse_error_message(sess):
             return error.text
 
     def do_login(self):
-        self.session = requests.Session()
+        # self.session = requests.Session()
```

Comment:
why?

Reply:
This got moved up to the constructor (see line 52) - should be deleted not a comment.

This whole refactor is pretty old now, I'll have to review it with `master` if it is going to stay - I never got the tests I wanted finished, so still in draft.